slim/nets/resnet_v2.py

```diff
@@ -115,7 +117,7 @@ def resnet_v2(inputs,
               global_pool=True,
               output_stride=None,
               include_root_block=True,
-              spatial_squeeze=True,
+              spatial_squeeze=False,
```

Comment:
What's the reason for converting all of these `spatial_squeeze` arguments to `False`?